Makefile

```diff
@@ -92,7 +92,7 @@ test_migrations: install-migrations-testing-requirements ## Tests migrations
 # Translations Handling
 #######################
 extract_translations: ## Extract strings and create source .po files
-	cd src/oscar; django-admin.py makemessages -a
+	cd src/oscar; django-admin.py makemessages -a --no-wrap
```

Comment:
I'd make this separate commit, since the change has nothing to do with the commit message "Update translatable strings.".

Reply:
OK, done.

Reply:
Can't see updates yet.